app/src/main/res/layout/control_main_list_item.xml

```diff
@@ -20,63 +20,66 @@
+                android:layout_marginStart="5dp"
                 android:text=""
                 android:textSize="16sp"
                 android:textStyle="bold"/>
 
             <Button
                 android:id="@+id/button_control_main1"
-                android:layout_width="25dp"
-                android:layout_height="25dp"
-                android:layout_marginStart="5dp"
-                android:layout_marginTop="22dp"
+                android:layout_width="0dp"
+                android:layout_weight="0.5"
+                android:layout_height="20dp"
                 android:background="@drawable/button_minus" />
 
             <EditText
-                android:id="@+id/edittext_control_main1"
-                android:layout_width="70dp"
+                android:id="@+id/edittext_control_main"
+                android:layout_width="0dp"
+                android:layout_weight="2.5"
                 android:layout_height="30dp"
-                android:layout_marginTop="25dp"
+                android:layout_marginTop="3dp"
                 android:gravity="center"
                 android:padding="@dimen/control_text_padding"
                 android:inputType="numberDecimal"
-                android:background="@drawable/control_edittext"/>
+                android:background="@drawable/control_edittext" />
 
             <Button
                 android:id="@+id/button_control_main2"
-                android:layout_width="25dp"
-                android:layout_height="25dp"
-                android:layout_marginTop="22dp"
+                android:layout_width="0dp"
+                android:layout_weight="0.5"
+                android:layout_height="20dp"
                 android:background="@drawable/button_plus" />
             <Button
                 android:id="@+id/button_control_main3"
-                android:layout_width="50dp"
+                android:layout_width="0dp"
+                android:layout_weight="1.5"
                 android:layout_height="35dp"
-                android:layout_marginTop="22dp"
+                android:layout_marginStart="5dp"
                 android:text="@string/set"
                 android:textColor="@color/white"
                 android:background="@color/colorPrimaryDark"/>
             <TextView
-                android:id="@+id/text_control_main2"
-                android:layout_width="110dp"
-                android:layout_height="30dp"
-                android:layout_marginStart="10dp"
-                android:layout_marginTop="22dp" />
+                android:id="@+id/tv_control_main2"
+                android:layout_width="0dp"
+                android:layout_weight="3"
+                android:layout_height="30dp" />
         </LinearLayout>
 
         <SeekBar
-            android:id="@+id/seekbar_control_main1"
+            android:id="@+id/seekbar_control_main"
             android:layout_width="match_parent"
             android:layout_height="50dp"
```

Comment:
What if we set this to `wrap_content`? That fixed the bubble rising above the line issue I had in my 480x800 screen :)